prefix-hex/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "prefix-hex"
```

Comment:
I wonder if that whole crate shouldn't be a feature PR to the `hex` crate instead. :thinking: I am sure you considered that, so why did you decide against it, and rather make a separate crate?

Reply:
Yeah we thought about that, but the Ethereum RPC API does a couple of things that probably wouldn't make it into `hex` anyway. So we didn't pursue this further.